python/ray/tune/execution/tune_controller.py

```diff
@@ -154,8 +154,20 @@ def _wrapped(self):
         return TrialRunnerWrapper(
             self,
             trial_executor=_FakeRayTrialExecutor(self),
-            runner_whitelist_attr={"search_alg", "get_trials", "_set_trial_status"},
-            executor_whitelist_attr={"has_resources_for_trial", "pause_trial", "save"},
+            runner_whitelist_attr={
```

Comment:
For the `TuneController`, I think we need to more keys:

```
"_schedule_trial_pause", "_schedule_trial_save", "_resource_updater"
```

The `FakeRayTrialExecutor` just passes through to these methods of the wrapped TuneController, which would still show the warning message.

Reply:
The `executor_whitelist_attr` also needs to whitelist `_tune_controller`.

Reply:
I don't think this is the case - the `FakeRayTrialExecutor` wraps around the actual TuneController, so it doesn't trigger any warning.

We also don't want to expose the tune controller attribute at all (the "real" executor won't have that property so we wouldn't want to use it anyway)

Reply:
Ah got it, my mistake!